pkg/router/router_test.go

```diff
@@ -93,6 +96,13 @@ func TestMain(m *testing.M) {
 	}
 	fmt.Printf("router working directory: %s\n", workdir)
 
+	h.workdir = workdir
+	h.dirs = map[string]string{
+		"whitelist": path.Join(workdir, "router", "whitelists"),
```

Comment:
We should use the filepath package:
```suggestion
		"whitelist": filepath.Join(workdir, "router", "whitelists"),
```